### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MilesTEG1**
Add information about the _**zigbee2mqtt**_  `configuration.yml` file because some like do the editing inside the home-assistant `configuration.yml` file.

### CODE REVIEW

File: docs/guide/adapters/README.md [hunk 1]
```diff
@@ -525,7 +525,7 @@ Initial development started on experimental (beta stage) support for Silicon Lab
 
     While the RCP firmware allows you to utilize the adapter with other protocols such as Thread, it requires offloading large parts of the application onto the host computer using an additional service, Zigbeed (Zigbee daemon) instead. This service is not currently included with Zigbee2MQTT/zigbee-herdsman and you will be required to install and maintain the dependency yourself. More discussion can be found [here](https://github.com/zigpy/zigpy/discussions/894).
 
-    If Zigbee2MQTT fails to start, try adding the following to your `configuration.yaml`
+    If Zigbee2MQTT fails to start, try adding the following to your zigbee2mqtt `configuration.yaml` (inside the `/config/zigbee2mqtt/` folder)
```
Comment (Koenkk) [inline docs/guide/adapters/README.md:528]: > (inside the `/config/zigbee2mqtt/` folder)

This is specific to HA addon, could you mention it there?

Comment (MilesTEG1) [reply docs/guide/adapters/README.md:528]: Yes indeed.
But I'm not sure how to do this 😢

Comment (Koenkk) [reply docs/guide/adapters/README.md:528]:
```
If Zigbee2MQTT fails to start, try adding the following to your Zigbee2MQTT `configuration.yaml` (for the Home Assistant addon inside the `/config/zigbee2mqtt/` folder)
```

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**